### PR TITLE
added lv_sjpg

### PR DESCRIPTION
**Pull request by vinodstanur**
@kisvegabor 
Capture

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+  - By default the image cache will be xres * 2 * 16 bytes (can be modified)
+  - Currently only 16 bit image format is supported (to do)
+# Sample code:
+```c
+lv_obj_t * img1;
+lv_obj_t * img2;
+
+void lv_demo_sjpeg(void)
+{
+    lv_jpeg_init();
```
Comment (kisvegabor) [inline README.md:19]: It should be called `lv_split_jpg_init()` to differentiate it from a "normal" jpg decoder. I'll rename the repo to `lv_lib_split_jpg` too.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,53 @@
-# _template
-Template repo with README and CI settings
+# LV SJPG DECODER 
+- (first commit)
+
+# Overview:
+  - SJPG is 'split-jpeg' which is a bundle of small jpeg fragments with an sjpg header.
```
Comment (kisvegabor) [inline README.md:5]: I'd add a line like "sjpg is a custom format based on the standard jpg" to be sure people will understand what it's about.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,53 @@
...
+lv_obj_t * img1;
+lv_obj_t * img2;
+
+void lv_demo_sjpeg(void)
+{
+    lv_jpeg_init();
+    img1 = lv_img_create(lv_scr_act(), NULL);
+    img2 = lv_img_create(lv_scr_act(), NULL);
+    lv_img_set_src(img1,  &wallpaper);            //image 1 from c array
+    lv_img_set_src(img2,  ".\\small_image.sjpg")  //image 2 from file system (fopen)
```
Comment (kisvegabor) [inline README.md:23]: Finally, we should use `lv_fs` to allow handling different file systems in a transparent way. [lv_fs_if](https://github.com/lvgl/lv_fs_if) has ready to use binding to C file functions.

**Review by kisvegabor — COMMENTED** [review]
Hi Vinod,

That's awesome! Thank you very much.

I've checked mainly the high-level API and added some comments.
Let me know what you think.

**Comment by amirgon** [comment]
Question:  
Will this project eventually allow loading "regular" JPG images, or only the special "sjpg" format?

File: README.md (hunk 1 above)
Comment (vinodstanur) [reply README.md:19]: Yes, lv_split_jpg_init makes more sense.

File: README.md (hunk 3 above)
Comment (vinodstanur) [reply README.md:23]: Okay, got it !

File: README.md (hunk 1 above)
Comment (kisvegabor) [reply README.md:19]: Okay, I renamed the repo.

File: README.md (hunk 2 above)
Comment (kisvegabor) [reply README.md:5]: Why was it marked as resolved?

**Comment by kisvegabor** [comment]
> Will this project eventually allow loading "regular" JPG images, or only the special "sjpg" format?

It's only for the special SJPG format. For normal JPG we need another "lib". 
Note that this special format still fills a great hole. Now we don't have a fast but small image format. E.g. PNG is also small but needs the whole image decompressed in RAM.

**Comment by kisvegabor** [comment]
I merge this PR as an initial code and we can add improvements later.

**Pull request merged**